settings.py

```diff
@@ -235,7 +236,7 @@
         'http://localhost:3000',
         'http://localhost:8000',
     )
-    STATICFILES_DIRS = [STATIC_ROOT]
+    STATICFILES_DIRS.append(STATIC_ROOT)
```

Comment:
It seems that the first element of `STATICFILES_DIRS` (which is 'ui/dist') doesn't have a full path while the second one (which is `STATIC_ROOT`) has. Is that correct?

Reply:
yes, it works fine that way